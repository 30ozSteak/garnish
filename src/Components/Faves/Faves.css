.faves-box {
  padding: 2rem;
  cursor: pointer;
}

.faves-box a:hover {
  color: black;
}
