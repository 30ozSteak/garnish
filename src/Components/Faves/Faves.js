import React from "react";
import { NavLink } from "react-router-dom";

import "./Faves.css";

const Faves = () => {
  return (
    <div className="faves-box">
      <NavLink to="/main">
        <h1> This one too </h1>
      </NavLink>
    </div>
  );
};

export default Faves;
