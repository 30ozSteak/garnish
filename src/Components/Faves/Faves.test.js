import React from "react";
import Faves from "./Faves";
import { shallow } from "enzyme";

describe("Faves", () => {
  it("should resemble the snapshot", () => {
    const wrapper = shallow(<Faves />);
    expect(wrapper).toMatchSnapshot();
  });
});
