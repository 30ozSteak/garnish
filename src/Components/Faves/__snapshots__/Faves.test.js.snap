// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faves should resemble the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Faves />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        aria-current="page"
        to="/main"
      >
        <h1>
           This one too 
        </h1>
      </NavLink>,
      "className": "faves-box",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "aria-current": "page",
        "children": <h1>
           This one too 
        </h1>,
        "to": "/main",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " This one too ",
        },
        "ref": null,
        "rendered": " This one too ",
        "type": "h1",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          aria-current="page"
          to="/main"
        >
          <h1>
             This one too 
          </h1>
        </NavLink>,
        "className": "faves-box",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <h1>
             This one too 
          </h1>,
          "to": "/main",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " This one too ",
          },
          "ref": null,
          "rendered": " This one too ",
          "type": "h1",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
